### DISCUSSION TITLE
Text selection in search results

### DISCUSSION THREAD

**Discussion by LuaKT** [community/community#8567] [Other Feature Feedback, Questions, & Ideas]
You can't currently select text in the search results, instead it just drags around like an image. It would be useful to be able to quickly select and copy snippets from the search results without having to open the full code file.

Thanks

**Comment by look** [comment]
Hi @LuaKT, thanks for the bug report! Can you give us some more details about what browser/OS you're using?

**Reply by LuaKT** [reply]
Hi @look, I'm using Firefox 94.0.2 on Linux. Just tested on Chrome and it doesn't have this issue.

Thanks

**Reply by LuaKT** [reply]
Left Firefox, right Chrome

https://user-images.githubusercontent.com/1272751/145309000-ce1bafa8-50a7-4587-8df7-d7f0bb7c40eb.mp4

**Reply by look** [reply]
Thanks for the great bug report. I replicated this in Firefox.

**Comment by drdaxxy** [comment]
_At least on Windows,_ **to select within link text, hold down the `Alt` key while pressing the left mouse button** _(then dragging the cursor or waiting a moment; if you let go immediately it's just interpreted as a click)_.

This works for selecting code in search results in Firefox. In other parts of the results page, it's a bit weird.

| Action | Firefox | Chrome |
| --- | --- | --- |
| Click-and-drag on code | ❌ | ✔ |
| Alt+click-and-drag on code | ✔ | ✔* |
| Click-and-drag on repo/file name | ❌ | ❌ |
| Alt+click-and-drag on repo/file name | 🟡† | ✔ |
| Click-and-drag on directory  | ❌† | 🟡‡ |
| Alt+click-and-drag on directory | ❌† | 🟡‡ |

*: Redundant.
†: Selects, but triggers click event on the start of the selection anyway. Over the repo name, clicking just opens a balloon so copying still works, anywhere else it navigates away.
‡: Selects, but *if the end of the selection is still over a link* (???) triggers click event on the start of the selection anyway.

(Firefox is current stable 95.0.2, Chrome is Microsoft Edge stable 96.0.1054.62, all on Windows 10.)
